### PR TITLE
conditional rendering first pass

### PR DESCRIPTION
**Pull request by tgberkeley**

### CODE REVIEW

File: docs/conditional_rendering.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+```
+
+```python eval
+docdemo_from(CondSimpleState, component=cond_simple_example)
+```
+
+
+## Negation
+
+You can use the logical operator `~` to negate a condition. In this example we show that by negating the condition `~CondNegativeState.show` within the cond, we then render the `rx.text("Text 1", color="blue")` component when the state var `show` is negative.
```
Comment (Alek99) [inline docs/conditional_rendering.md:59]: Instead of negation this section should have a general title for all var operation and link to the var operation docs as reference

File: docs/conditional_rendering.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+        self.b = random.randint(0, 10)
+        self.c = random.randint(0, 10)
+
+
+def cond_nested_example():
+    return rx.vstack(
+        rx.button("Toggle", on_click=CNS.change),
+        rx.text(f"a: {CNS.a}, b: {CNS.b}, c: {CNS.c}"),
+        rx.cond(
+            ((CNS.a > CNS.b) & (CNS.a > CNS.c)) & ((CNS.a != CNS.b) & (CNS.a != CNS.c)),
```
Comment (Alek99) [inline docs/conditional_rendering.md:216]: The combination of nesting and 4 conditionals check is making this example to hard to follow, a more simple example of nesting would make it clearer

**Review by Alek99 — CHANGES_REQUESTED** [review]

File: docs/conditional_rendering.md (hunk 2 above)
Comment (tgberkeley) [reply docs/conditional_rendering.md:216]: created a new example above it that is much simpler

Comment (Alek99) [reply docs/conditional_rendering.md:269]: Nice looks good

**Review by Alek99 — APPROVED** [review]
